csrc/scheduler/pointwise_non_tma.h

```diff
@@ -0,0 +1,22 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <fusion.h>
+#include <scheduler/pointwise.h>
+
+namespace nvfuser {
+namespace pointwise_non_tma {
```

Comment:
nit: Our naming scheme of namespaces isn't really consistent, but I'd say this name at the top level sounds a little too specific. Why not `nvfuser::pointwise::non_tma` and also `nvfuser::pointwise::tma`?